jira.js

```diff
@@ -123,7 +123,45 @@ router.post('/webhook', async (req, res) => {
 			link.finished = true;
 			link.save();
 		} else if (req.body.transition && req.body.transition.transitionName === 'Send to Ikari') {
-			clipRequest(['youtube', req.body.issue.fields.customfield_10200, req.issue.fields.customfield_10201, req.issue.fields.summary, 'mkv']);
+			const videoRegex = /^(http(s)?:\/\/)?(www\.)?youtu((\.be\/)|(be\.com\/watch\?v=))[0-z]{11}$/g;
+			let videoType = 'youtube';
+			if (req.body.issue.fields.customfield_10200.match(videoRegex) === null) {
```

Comment:
```suggestion
			if (!videoRegex.test(req.body.issue.fields.customfield_10200)) {
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test